### PR TITLE
Added documentation for product shadow

### LINKED ISSUE #357: Documentation of Digital Shadow
The Digital Shadow (issue #297) needs to be documented. It will be in the file .../Concepts/Digital Shadow.
Things to be documented:
- Data structures (different types of ID's, relevant timescaleDB tables etc.)
- General Overview over the functionality
- Description of implementation guidelines
- functions and limitations
- planned features
- etc.

In this documentation we are showing general concepts regarding the digital shadow. Not yet fully specified features will be discussed in separate issues, please look them up in the superordinate issue #297)

### PR DESCRIPTION
**Pull request by larswitte**
Fixed #357

### CODE REVIEW

File: docs/content/en/docs/Concepts/Digital Shadow/index.md [hunk 1]
```diff
@@ -0,0 +1,418 @@
+
+---
+title: "Digital Shadow - track and trace"
+linkTitle: "Digital Shadow - track and trace"
+weight: 2
```
Comment (JeremyTheocharis) [inline docs/content/en/docs/Concepts/Digital Shadow/index.md:5]: You can remove the weight here

File: docs/content/en/docs/Concepts/Digital Shadow/index.md [hunk 2]
```diff
@@ -0,0 +1,418 @@
...
+description: >
+  A system of features allowing tracking and tracing of individual parts through the production process. 
+---
+TEMP: [count](#count)
```
Comment (JeremyTheocharis) [inline docs/content/en/docs/Concepts/Digital Shadow/index.md:9]: Should be removed :)

File: docs/content/en/docs/Concepts/Digital Shadow/index.md [hunk 3]
```diff
@@ -0,0 +1,418 @@
+
```
Comment (JeremyTheocharis) [inline docs/content/en/docs/Concepts/Digital Shadow/index.md:1]: Very detailed guide! Perfect!

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]

**Review by HW138 — APPROVED** [review]

**Comment by larswitte** [comment]
Please merge, it's finished.

**Pull request merged**